tests/test_sandbox_fork_exc_leak.py

```
@@ -0,0 +1,195 @@
+    @pytest.fixture()
+    def shell_manager(self):
+        from nemo_skills.code_execution.local_sandbox.local_sandbox_server import ShellManager
+
+        mgr = ShellManager()
+        yield mgr
+        # Cleanup all shells
+        for sid in list(mgr.shells.keys()):
+            try:
+                mgr.stop_shell(sid)
+            except Exception:
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)